velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -865,6 +867,69 @@ RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(RowVectorPtr rowVector) {
       std::move(children));
 }
 
+// Utility function to check if a RowVector can have nested lazy children.
+bool canWrapChildrenInLazy(const VectorPtr& baseVector) {
```

Comment:
nit: canHaveRowChildInLazy(...)

Reply:
changed it to canWrapRowChildInLazy, let me know if this works